lib/modules/manager/github-actions/extract.ts

```diff
@@ -68,6 +69,48 @@ export function extractPackageFile(content: string): PackageFile | null {
       deps.push(dep);
     }
   }
+  return deps;
+}
+
+function extractContainer(container: string | Container): PackageDependency {
+  let dep: PackageDependency;
+  if (typeof container === 'string') {
+    dep = getDep(container);
+  } else {
+    dep = getDep(container?.image);
+  }
+  return dep;
+}
+
+function extractWithYAMLParser(content: string): PackageDependency[] {
+  logger.trace('github-actions.extractWithYAMLParser()');
+  const deps: PackageDependency[] = [];
+
+  const pkg = load(content, { json: true }) as Workflow;
```

Comment:
Doesn't this function throw if it's invalid YAML? If so, what's the impact? Do we log warnings or errors, and does it cause the whole file extraction to now fail, even if before the regex worked?

Reply:
you are right. @maxbrunet Add a try catch around the load.